### ISSUE TITLE
MapleStory Not Working (216150)

### ISSUE THREAD

**Issue by arnonsur** [ValveSoftware/Proton#1491]
# Compatibility Report
- Name of the game with compatibility issues: MapleStory
- Steam AppID of the game:216150

## System Information
- GPU: Intel(R) HD Graphics 520 (Skylake GT2) x86/MMX/SSE2
- Driver/LLVM version: 3.0 Mesa 18.0.5
- Kernel version: 5.15.0-34-generic
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/arnonsur/ff101482790483f1dc8dca2eaa0cc9bb
- Proton version: 3.7-6

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-216150.log](https://github.com/ValveSoftware/Proton/files/2387019/steam-216150.log)

## Symptoms 
When I launch the game I am prompted to put my Nexon account information but I can't do it as the email box doesn't respond to me clicking on it. The text is all messed up (I only know it's the email box because I've seen it on windows). That's even before getting to the game itself, which I heard from others doesn't work, but haven't personally been able to reach that point as I can't get passed that initial window.

## Reproduction
Just launch the game and you'll see.

**Comment by katanakiwi** [comment]
It is very, very unlikely that MapleStory will ever work with Proton due to the heavy anti-cheat software used.

**Comment by SolarLune** [comment]
Same story with MapleStory 2, I'm guessing, then?

**Comment by katanakiwi** [comment]
> Same story with MapleStory 2, I'm guessing, then?

I have no clue if that uses any anti-cheat programs. If they do, and don't intend to release an OSX or native version, chances are slim it'll ever work.

**Comment by jvschiavo** [comment]
I'm sorry to be off-topic, but does anyone know why MapleStory (the first one) isn't available on my region (Brazil)? I can't seem to find the reason on Google. D:

**Comment by SolarLune** [comment]
@katanakiwi Yeah, it uses Nexon Game Security. I was just saying that since it also has anti-cheat, it's a similar story. Thanks for the response~

@jvschiavo No ideas on my side~

**Comment by skytetsu** [comment]
i look the log, seems in loop, after the look the security cookie and the c++ runtime the app start locating and relocating the files, adding the dll files of c++ maybe gives the security cookie and maple start runing? just a wild guess, other parts in the log say of kernel 32 bits i guess is windows kernel. can be that found in wine? if not is really a lost case.
https://github.com/zfigura/wine/tree/esync can be used in dll libraries, in debian maybe a issue because is force to used the alternatives of c++ files, so cannot used the dll files
readme of DXVK=>
Troubleshooting
DXVK requires threading support from your mingw-w64 build environment. If you are missing this, you may see "error: 'mutex' is not a member of 'std'". On Debian and Ubuntu, this can usually be resolved by using the posix alternate, which supports threading. For example, choose the posix alternate from these commands (use i686 for 32-bit):

update-alternatives --config x86_64-w64-mingw32-gcc
update-alternatives --config x86_64-w64-mingw32-g++
<===

**Comment by romulasry** [comment]
Same as of today with Proton Experimental.

**Comment by eyalmazuz** [comment]
Game the works with proton experimental but still unplayable
you can lunch the game and go to the character selection,
it's better to hit alt-enter before selecting a character and switching to windowed mode
since from my experience when choosing a character in fullscreen mode makes the computer freeze (and I had to reboot to get control back)

but in windowed mode, you can enter the game and even play for a minute or two before it freezes for a minute or so and then crash
I'm attaching the log of the game here using ``PROTON_LOG=1 %command%`` 

[steam-216150.log](https://github.com/ValveSoftware/Proton/files/8501233/steam-216150.log)

**Comment by solenum** [comment]
Any update on this one?  Seeing mostly borked on protondb but seems no real update here for 2 years

**Comment by Luquatic** [comment]
Game works with some tweaks that this repo uses: https://github.com/oldschoola/linux_maplestory
Can't this be implemented into Proton itself?
